products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+releasePolicyLink: https://www.sqlite.org/news.html
+changelogTemplate: https://sqlite.org/releaselog/__LATEST__/.html
```

Comment:
This generate wrong links, such as https://sqlite.org/releaselog/2.00.08/.html.

Reply:
Yes, it's because I did add trailing `0` for better sorting... I remove them :zap:

Reply:
There is this page : https://www.sqlite.org/draft/chronology.html but some tas are missing too... :sweat:

Reply:
... and we cannot fin them on https://github.com/sqlite/sqlite/tags ...

Reply:
It does not matter if some links are absent, just use `link: null` in that case.

Reply:
Used https://www.sqlite.org/changes.html instead. It contains all the versions with anchors.